### PR TITLE
las: add a safeguard for two bytes color detection

### PR DESCRIPTION
**Pull request by zarov**
I forgot to add this in #927, sorry about that !

### CODE REVIEW

File: modules/las/src/lib/parse-las.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ function detectTwoByteColors(colorDepth, decoder, batchSize) {
       twoByteColor = true;
       break;
     case 'auto':
+      if (!decoder.getPoint(0).color) break;
```
Comment (ibgreen) [inline modules/las/src/lib/parse-las.js:15]: Just nits (which means fix not required)

I would have thought the linter should have complained here. We usually require braces around if statement clauses and also we typically disallow extra breaks and continues. I wonder if linter is disabled here?

Wrapping this case in `if (decoder.getPoint(0).color) {` would have been more in line with the existing style.

**Review by ibgreen — APPROVED** [review]

File: modules/las/src/lib/parse-las.js (hunk 1 above)
Comment (zarov) [reply modules/las/src/lib/parse-las.js]: Noted and fixed !

**Review by ibgreen — APPROVED** [review]

**Comment by ibgreen** [comment]
lint failure, you'll need to add an `// eslint-disable-next-line max-depth`

Sorry for the churn. Otherwise ready to merge.

**Comment by zarov** [comment]
Fixed, sorry for this. Why does the max depth is limited to 3 here ? Last time I made changes, it was 12 I think, which seems okay, but isn't 3 too small ?

**Comment by ibgreen** [comment]
> Fixed, sorry for this. Why does the max depth is limited to 3 here ? Last time I made changes, it was 12 I think, which seems okay, but isn't 3 too small ?

Hmm, not sure I think max-depth is 4 by default in the linter rules we use.

**Pull request merged**